crates/deploy-contracts-bin/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
```

Comment:
nit: I find the `-bin` in the crate name a bit redundant. Maybe put the crate under `crates/tools/*` to make it easier to not include these in the default workspace members to not slow down builds.